src/scores/processing/cdf/cdf_functions.py

```diff
@@ -310,8 +315,11 @@ def decreasing_cdfs(cdf: xr.DataArray, threshold_dim: str, tolerance: float) ->
     This is sometimes violated due to rounding issues or bad forecast process.
     `decreasing_cdfs` checks CDF values decrease beyond specified tolerance; that is,
     whenever the sum of the incremental decreases exceeds tolerarance.
+
     For example, if the CDF values are
-        [0, 0.4, 0.3, 0.9, 0.88, 1]
+
+    `[0, 0.4, 0.3, 0.9, 0.88, 1]`
+
```

Comment:
I think this is clearer if it's not broken out, but is part of one paragraph (subjective opinion again)

Reply:
Thanks, fixed.